Android/AmazonConnectInAppCallingAndroidSample/app/src/main/res/layout/config_dialog.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This feature addition could be added to commit message or PR description.

Reply:
added